FordLabs/PeopleMover
# 230 Fading New Badge

## Issue
Resolves 230 - make the New Badge fade over time

## What was done
- [x] Add new column to person table in DB to hold date for when new checkbox gets marked
- [x] Update Person in backend to read/write new field from/to DB
- [x] Update Person in frontend to have new field
- [x] In frontend, set new field on Person when'New' checkbox is checked
- [x] In frontend, set style of NewBadge component based on date difference between viewing date and new field

## How to test
1. In PeopleMover, create a person and mark them as new. Use Calendar to move viewing date to view how the NewBadge style changes as you move from 7 to 8 days after marking new, and 15 to 16 days after marking new. 
2. New can be used in other locations. It should be the standard style (1-7 days, darker) for all other usages.
